filesystem/dirs_test.go

```diff
@@ -14,21 +14,19 @@ import (
 var RootFolder = "/"
 
 func TestPathExists(t *testing.T) {
-	tempFile := filepath.Join(os.TempDir(), "testdir"+uuid.New().String()+"_"+t.Name())
+	tempFile := filepath.Join(t.TempDir(), uuid.New().String()+"_"+t.Name())
 	_, err := os.Create(tempFile)
 	require.NoError(t, err, "couldn't create temp path")
 	assert.True(t, PathExists(tempFile), "expecting existence of path")
-	os.RemoveAll(tempFile)
 }
 
 func TestGetFullDirectoryPath(t *testing.T) {
-	tempFile := filepath.Join(os.TempDir(), "testdir"+uuid.New().String()+"_"+t.Name())
+	tempFile := filepath.Join(t.TempDir(), "testdir"+uuid.New().String()+"_"+t.Name())
```

Comment:
I don't get why this has to be that complicated. It could be just:
```suggestion
	tempFile := filepath.Join(t.TempDir(), "test_file")
```

Btw, looking at the test it's not actually treated as a path to a **file**, but a directory. So:
```suggestion
	directory := filepath.Join(t.TempDir(), "test_directory")
```